docs/DeveloperGuide.md

```diff
@@ -695,3 +695,77 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Mass Operations [Proposed]
+
+#### What is the feature about
+Provides a way for users to mark and unmark multiple `Task` at a time. The current implementation of mark and unmark
+users have to mark and unmark multiple tasks as completed or uncompleted individually, one at a time.
```

Comment:
Maybe for this statement, you can use more connectors to make the sentence flow better?
> **For** the current implementation of `mark` and `unmark`**,** users have to mark and unmark .....